services/core-api/app/api/now_applications/resources/now_application_delay_resource.py

```diff
@@ -48,7 +48,16 @@ def post(self, now_application_guid):
         now_delay = NOWApplicationDelay._schema().load(request.json)
         now_app.application_delays.append(now_delay)
 
-        ##ensure this starts after most recent edit
+        # update now status
+        if now_app.now_application is not None and now_delay.delay_type_code is not None and now_delay.delay_type_code == "OAB":
```

Comment:
if `now_app.now_application` is None it will go into the else block and get a null reference exception. Is this intentional?

Reply:
good catch, not intentional. I will add another if statement

Reply:
fixed.